src/App.css

```diff
@@ -1,17 +1,28 @@
 .mainContainer {
   width: 100%;
   height: 100%;
-  background: black;
+  background: white;
   position: absolute;
   overflow: hidden;
   user-select: none;
 }
+
 .bg {
   height: 100%;
   width: 100%;
   position: absolute;
   top: 0;
 }
+
+.collisionBox {
+  position: absolute;
+  left: 11px;
+  top: 30px;
```

Comment:
Shift all these numbers to JS side and inject from there. This will change with screen size